Source Packages/io/rerum/crud/tinyOverwrite.java

```
@@ -0,0 +1,167 @@
+            connection.setDoOutput(true);
+            connection.setDoInput(true);
+            connection.setRequestMethod("PUT");
+            connection.setUseCaches(false);
+            connection.setInstanceFollowRedirects(true);
+            connection.setRequestProperty("Content-Type", "application/json; charset=utf-8");
+            connection.setRequestProperty("Authorization", "Bearer "+pubTok);
+            connection.connect();
+            try{
+                DataOutputStream out = new DataOutputStream(connection.getOutputStream());
+                //Pass in the user provided JSON for the body of the rerumserver v1 request
+                out.writeBytes(requestJSON.toString());
+                out.flush();
+                out.close(); 
+                codeOverwrite = connection.getResponseCode();
+                BufferedReader reader = new BufferedReader(new InputStreamReader(connection.getInputStream(),"utf-8"));
+                while ((line = reader.readLine()) != null){
+                    //Gather rerum server v1 response
+                    sb.append(line);
+                }
+                reader.close();
+            }
+            catch(IOException ex){
+                //Need to get the response RERUM sent back.
+                BufferedReader error = new BufferedReader(new InputStreamReader(connection.getErrorStream(),"utf-8"));
+                String errorLine = "";
+                while ((errorLine = error.readLine()) != null){  
+                    sb.append(errorLine);
+                } 
+                error.close();
+            }
+            connection.disconnect();
+            response.setStatus(codeOverwrite);
+            response.getWriter().print(sb.toString());
+        }
+        
+    }
+
+    // <editor-fold defaultstate="collapsed" desc="HttpServlet methods. Click on the + sign on the left to edit the code.">
+    /**
```

There is a PR that says we shouldn't support incorrect methods, maybe get rid the doGet here and let a GET /overwrite fail hard.